src/components/media-player/media-player.component.spec.ts

```diff
@@ -46,31 +43,31 @@ fdescribe('Media Player Component - Audio', () => {
-        component.displayName = '';
+        component.audioSource = 'data:audio/mpeg;base64,/+MYxAAAAANIAAAAAExBTUUzLjk4LjIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+        component.filename = '';
         fixture.detectChanges();
-        expect(getTitle().innerText).toBeFalsy();
+        expect(getTitle()).toBeFalsy();
     });
 
-    it('should display displayName when one is added to the markup', () => {
-        component.displayName = 'Display Name';
+    it('should display Display Name when one is added to the markup', () => {
+        component.filename = 'Display Name';
         fixture.detectChanges();
-        expect(getTitle().innerText).toBeTruthy();
+        expect(getTitle()).toBe('Display Name');
     });
 
     it('should display the filename when a file is used', (done: DoneFn) => {
-        component.displayName = '';
+        component.filename = '';
         component.audioSource = 'https://uxaspects.github.io/UXAspects/assets/Ocean-Waves.mp3';
         fixture.detectChanges();
 
         setTimeout(() => {
             fixture.detectChanges();
-            expect(getTitle().innerText).toBe('Ocean-Waves.mp3');
+            expect(getTitle()).toBe('Ocean-Waves.mp3');
             done();
         }, 2000);
     });
 
-    function getTitle(): any {
-        return nativeElement.querySelector('.player-container .audio-player .audio-file-name');
+    function getTitle(): string {
+        return (<HTMLElement>nativeElement.querySelector('.player-container .audio-player .audio-file-name')).innerText;
```

Comment:
The `querySelector` function accepts a generic to specify it's type, which means we don't need to cast it, for example, this could be:

```typescript
return nativeElement.querySelector<HTMLParagraphElement>('...').innerText;
```